waku/v2/node/wakunode2_no_rln.go

```diff
@@ -5,6 +5,10 @@ package node
 
 import "context"
 
+// SupportsRLN indicates if go-waku was compiled with RLN support or not
+const SupportsRLN = false
```

Comment:
Since supporting RLN is a property of WakuNode, wouldn't it be better to have this as a variable in the WakuNode struct and set the value from these 2 files rather than a const.

Reply:
RLN support in this case means if it's compiled with RLN support, but it does not indicate if it is enabled or not (which happens with `--rln-relay` flag, which should be available via `w.opts.rlnEnabled` iirc).  Having it as a constant would mean no possibility of anyone updating this value by mistake during runtime.

Reply:
Ok, but do we still want to provide that flag? 
I remember @alrevuelta mentioning that RLN should be enabled by default for all pubSub topics.
Does it mean this flag itself will have to be removed?

Reply:
if `--rln-relay` is passed, it means that RLN will be applied to all pubsub topics. Before https://github.com/waku-org/go-waku/pull/663, it was required to pass `--rln-relay-pubsub-topic` and `--rln-relay-content-topic` and rln proof verification would be applied exclusively to messages that were sent on that pubsub topic with that content topic.